addon/components/bs-form/element.js

```diff
@@ -7,8 +7,8 @@ export default class BsFormElementWithChangesetValidationsSupport extends BsForm
 
   @dependentKeyCompat
   get errors() {
-    let error = get(this, `model.error.${this.property}.validation`);
-    return error ? [error] : [];
+    let errors = get(this, `model.error.${this.property}.validation`);
+    return errors ? errors : [];
```

Comment:
If I understand the documentation of validated-changeset correctly, `model.error.${this.property}.validation` could be either a string or an array: https://github.com/validated-changeset/validated-changeset/#error

Ember Bootstrap expects `errors` property of FormElement to be an array of validation messages: https://www.ember-bootstrap.com/api/classes/Components.FormElement.html#property_errors

Of course `model.error.[this.property]` could also be undefined if the property is valid. So it seems like we need to handle three cases here:

1. `model.error.${this.property}.validation` is `undefined` -> return empty array
2. `model.error.${this.property}.validation` is a string -> wrap that string into an array and return that one
3. `model.error.${this.property}.validation` is an array -> return that array

Can you please update the code accordingly? Some in-source comments about the different states may also be helpful for future readers.